napari/_qt/qt_main_window.py

```diff
@@ -874,11 +1025,29 @@ def activate(self):
         self._qt_window.raise_()  # for macOS
         self._qt_window.activateWindow()  # for Windows
 
+    def _toggle_theme(self):
```

Comment:
is this necessary?  the old method would have called `viewer._toggle_theme`, which would have emit a `viewer.theme` event. which is already connecting to `_update_theme`, why do we need another method?

Reply:
You are right, I think before some of the changes you made in another PR, I was not getting the right behavior. It does now. Thanks!